spec/api-browser-window-spec.js

```diff
@@ -94,12 +94,19 @@ describe('browser-window module', function() {
       });
       return w.loadURL('about:blank');
     });
-    return it('should emit did-fail-load event', function(done) {
+    it('should emit did-fail-load event', function(done) {
```

Comment:
Should this message be updated now to be more specific for which failure case it is testing?

Like `'should emit did-fail-load-event for files that do not exist'` or `'should emit did-fail-load-event for URLs that 404'`?